src/test/java/kg/jarkyn/server/ReceivingSocketTest.java

```diff
@@ -0,0 +1,62 @@
+package kg.jarkyn.server;
+
+import org.junit.Test;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+import static org.junit.Assert.assertTrue;
+
+public class ReceivingSocketTest {
```

Comment:
I kept it to show Uku that I tried, should have mentioned this in the comment. This is not staying past this, will try and make sure it doesn't end up in master's history